unittest/gunit/segfault-t.cc

```diff
@@ -63,26 +63,10 @@ TEST_F(FatalSignalDeathTest, Segfault) {
    gtest library instead.
   */
   EXPECT_DEATH_IF_SUPPORTED(*pint = 42, "");
-<<<<<<< HEAD
-#elif defined(__SANITIZE_ADDRESS__)
+#elif defined(HAVE_ASAN)
   /* AddressSanitizer */
   EXPECT_DEATH_IF_SUPPORTED(*pint = 42, ".*ASAN:(DEADLYSIGNAL|SIGSEGV).*");
```

Comment:
The conflict resolutions does not look right taking into account the fix from Oracle
https://github.com/mysql/mysql-server/commit/2ae0ca1ffe239e36995f4516290d8963ff434241
Please, double check with gcc-9/gcc-10 ASan